glib/src/gstring.rs

```diff
@@ -694,10 +694,17 @@ impl GStringPtr {
     // rustdoc-stripper-ignore-next
     /// Returns the corresponding [`&str`].
     #[inline]
-    pub fn to_str(&self) -> &str {
+    pub fn as_str(&self) -> &str {
         self.to_gstr().as_str()
     }
 
+    // rustdoc-stripper-ignore-next
+    /// This is just an alias for [`as_str`].
+    #[inline]
+    pub fn to_str(&self) -> &str {
```

Comment:
Should we mark this one as deprecated ?

Reply:
I'd vote we add some sort of comment that signals it should be removed in the next semver bump instead.

There's really no urgency to make people port.

Reply:
I'd deprecate it to make sure people notice this. Deprecations are only warnings and can easily be ignored. I can guarantee you that nobody will otherwise port before the next breaking release :)

Reply:
OK done